atom/index.d.ts

```diff
@@ -1,5 +1,7 @@
 import { lastAction } from '../action/index.js'
 
+type AllKeys<T> = T extends any ? keyof T : never
```

Comment:
Moved here because we need it here, and `atom` is imported in `map`, where this definition was before